recipes/orjson/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "orjson" %}
+{% set version = "2.6.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+    sha256: 910796567d46223a7f4eaea0cc2cb4d3d3adb7a6bccc5ffa8fa2f95b4296d8d9
+    # Lucky for us, this doesn't require glib >2.12
+  - url: https://static.rust-lang.org/dist/2020-04-26/rust-nightly-x86_64-unknown-linux-gnu.tar.gz  # [linux]
+    sha256: a723247878b5b65ec511c86108b9adb99bd1fa9b78ed87986283b93bf9e50d20  # [linux]
+    folder: rust-nightly
+  - url: https://static.rust-lang.org/dist/2020-04-26/rust-nightly-x86_64-apple-darwin.tar.gz  # [osx]
+    sha256: 9f8d7ee88c596465336297aa45c8f9b8d06480728e57b99bb2bc29371d518758  # [osx]
+    folder: rust-nightly
+  - url: https://static.rust-lang.org/dist/2020-04-26/rust-nightly-x86_64-pc-windows-msvc.tar.gz  # [win64]
+    sha256: c6fd886231faa52552757da7a1d813b66aa02413218343089008751fd1b56df7  # [win64]
+    folder: rust-nightly
```

Comment:
For windows, let's try the following in bld.bat:
```
FOR /F "delims=" %%i in ('cygpath.exe -u "%SRC_DIR%\rust-nightly-install"') DO set "pfx=%%i"
bash %SRC_DIR%\rust-nightly\install.sh --verbose --prefix=%pfx% --disable-ldconfig
set PATH=%SRC_DIR%\rust-nightly-install\bin:%PATH%
maturin build --no-sdist --release --strip --manylinux off
%PYTHON% -m pip install . -vv
```

And add: `- posix  # [win]` to the build requirements.

Reply:
Implemented. Lets see if it works

Reply:
Failed. Do we need to change the path from the rust nightly path back to the pip src_dir before doing pip install?